lua/adr/init.lua

```diff
@@ -61,11 +65,11 @@ end
 M.write_to_path = function(template_name, file_name)
     local template = vim.fn.readfile(template_name)
 
-    if vim.fn.isdirectory(vim.fn.getcwd() .. '/docs') == 0 then
-        vim.fn.mkdir(vim.fn.getcwd() .. '/docs')
+    if vim.fn.isdirectory(vim.fn.getcwd() .. M.config.doc_dir) == 0 then
```

Comment:
`vim.fn.getcwd()` returns the directory without a trailing slash. Due to the proposed change of configuring a relative path without the leading slash, we need to add it either in each place, where we use `M.config.doc_dir` or in the `setup` function.